packages/website/components/LiveEditor/ComponentSource.tsx

```diff
@@ -59,13 +60,13 @@ const styles = {
   `,
   copyButton: css`
     position: absolute;
-    top: ${tokens.spacingS};
+    bottom: ${tokens.spacingS};
```

Comment:
moved it to the bottom, to not overlap the content (at the bottom most likely we have closing tags, so less probably that buttons will cover the content)